src/evict/evict_page.c

```diff
@@ -574,9 +592,9 @@ __evict_review(
 	 * table, allowing the eviction of pages we'd otherwise have to retain
 	 * in cache to support older readers.
 	 */
-	if (ret == EBUSY && lookaside_retry && __wt_cache_stuck(session)) {
-		LF_CLR(WT_EVICT_SCRUB | WT_EVICT_UPDATE_RESTORE);
-		LF_SET(WT_EVICT_LOOKASIDE);
+	if (ret == EBUSY && lookaside_retry) {
+		LF_CLR(WT_REC_SCRUB | WT_REC_UPDATE_RESTORE);
+		LF_SET(WT_REC_LOOKASIDE);
```

Comment:
Why did we stop checking `cache->evict_aggressive_score`?

Reply:
It's checked above via the `_HARD` flags, before setting `lookaside_retryp`.